app/views/shared/_navbar.html.erb

```diff
@@ -2,7 +2,7 @@
   <%= link_to root_path do %>
     <%= image_tag "artify_logo", class: "logo" %>
   <% end %>
-  <%= link_to "rent a piece" arts_path %>
+  <%= link_to "rent a piece", arts_path %>
   <%= render "shared/form" %>
 
   <%# Dropdown avatar %>
```

Comment:
This as already been fixed and merged https://github.com/wJoenn/rentart/pull/43

You can remove it from your PR.
If you want to work on an issue don't forget to assign yourself to it in github AND make a seperate PR just for that next time :)